test/components/Profile/ProfileComponent.test.jsx

```diff
@@ -4,37 +4,17 @@ import { cleanup, render, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { afterEach, describe, expect, it, vi } from 'vitest';
 import { ProfileComponent } from '@components/Profile';
-import { SignedInUserContext } from '@contexts';
-import * as profileHelper from '../../../src/model-helpers/Profile';
 import '@testing-library/jest-dom/extend-expect';
 import createMatchMedia from '../../helpers/createMatchMedia';
 
```

Comment:
Seems like there's one more extra line that can be removed

Reply:
Thanks! Removed